livekit-agents/livekit/agents/stt/stt.py

```diff
@@ -47,8 +47,14 @@ class STTCapabilities:
 
 
 class STT(ABC):
-    def __init__(self, *, capabilities: STTCapabilities) -> None:
+    def __init__(
+        self,
+        *,
+        capabilities: STTCapabilities,
+        timeout: float = 0,
```

Comment:
Should we use ` timeout: float = 0`, or  `timeout: float | None = None`
cc @livekit/agent-devs for preferences, I know python use float | None for some of his methods (like asyncio.wait_for). So I would vote to do what Python is doing

Reply:
let's change to `timeout: float | None = None`